data/json/recipes/tools/tools_hand.json

```diff
@@ -81,7 +81,7 @@
   {
     "type": "recipe",
     "activity_level": "BRISK_EXERCISE",
-    "result": "swage",
+    "result": "hotcut",
```

Comment:
I think you missed this one, you should have a recipe for both a swage and a hotcut, not replace the swage recipe with hotcut.